Missionframework/presets/init_presets.sqf

```diff
@@ -318,4 +321,6 @@ zeropos = [0,0,0];
 KPLIB_sarWreck = "Land_Wreck_Heli_Attack_01_F";
 KPLIB_sarFire = "test_EmptyObjectForFireBig";
 
+KPLIB_initPresets = true;
```

Comment:
Where is this used or is it a preparation for future implementations?

Reply:
As discussed in Discord this can be used for debug/"for future implementations"